src/Join.tsx

```diff
@@ -0,0 +1,68 @@
+import type { ReactElement } from "react";
+import type { Socket } from "socket.io-client";
+
+import { StyleSheet, css } from "aphrodite";
+import useFormActions from "useFormActions";
+import Button from "./common/Button";
+import TextInput from "./common/TextInput";
+
+type JoinProps = Readonly<{
+  socket: Socket;
+  onClick: () => void;
+  onChange: () => void;
```

Comment:
take a look at your component definition, you're expecting 3 props: socket, onclick, and onchange but you're only using socket. Don't add unnecessary props